lxd/network/driver_bridge.go

```diff
@@ -1502,6 +1505,20 @@ func (n *bridge) setup(oldConfig map[string]string) error {
 			if err != nil {
 				return err
 			}
+
+			if len(remotes) > 1 {
+				for _, remote := range remotes {
+					if _, err := shared.RunCommand(
+						"bridge", "fdb",
+						"append", "00:00:00:00:00:00",
+						"dev", tunName,
+						"dst", strings.TrimSpace(remote),
```

Comment:
No need for this when using `shared.SplitNTrimSpace(args[1], ",", -1, false)`

Reply:
Done!